samples/generate-offline-map-with-custom-parameters/src/main/java/com/esri/arcgismaps/sample/generateofflinemapwithcustomparameters/screens/GenerateOfflineMapWithCustomParametersScreen.kt

```diff
@@ -0,0 +1,315 @@
+                if (mapViewModel.showJobProgressDialog) {
+                    JobLoadingDialog(title = "Generating offline map...",
+                        progress = mapViewModel.offlineMapJobProgress,
+                        cancelJobRequest = { mapViewModel.cancelOfflineMapJob() })
+                }
+
+                // Display a dialog if the sample encounters an error
+                mapViewModel.messageDialogVM.apply {
+                    if (dialogStatus) {
+                        MessageDialog(
+                            title = messageTitle, description = messageDescription, onDismissRequest = ::dismissDialog
+                        )
+                    }
+                }
+                if (mapViewModel.showResetButton) {
+                    Button(
+                        onClick = {
+                            mapViewModel.reset()
+                        }, modifier = Modifier.align(Alignment.CenterHorizontally)
+                    ) {
+                        Text("Reset to online map")
+                    }
+                }
+            }
+        },
+        // Floating action button to show the parameter overrides bottom sheet
+        floatingActionButton = {
+            if (!showBottomSheet && !mapViewModel.showResetButton) {
+                FloatingActionButton(modifier = Modifier.padding(bottom = 36.dp, end = 12.dp),
+                    onClick = { showBottomSheet = true }) {
+                    Icon(
+                        Icons.Filled.Settings, contentDescription = "Parameter overrides"
+                    )
+                }
+            }
+        })
+}
+
+@Composable
+fun OverrideParameters(
+    defineParameters: (Int, Int, Int, Boolean, Boolean, Int, Boolean) -> Unit,
+    setBottomSheetVisibility: (Boolean) -> Unit
+) {
+
+    // Collection of parameter overrides to set in this composable
+    var minScale by remember { mutableFloatStateOf(15f) }
+    var maxScale by remember { mutableFloatStateOf(20f) }
+    var extentBufferDistance by remember { mutableFloatStateOf(150f) }
+    var includeSystemValves by remember { mutableStateOf(false) }
+    var includeServiceConnections by remember { mutableStateOf(false) }
+    var minHydrantFlowRate by remember { mutableFloatStateOf(500f) }
+    var cropToWaterPipeExtent by remember { mutableStateOf(false) }
+
+    Column(
+        modifier = Modifier
+            .wrapContentSize()
+            .background(MaterialTheme.colorScheme.background)
+            .padding(12.dp)
+            .verticalScroll(rememberScrollState()),
+        horizontalAlignment = Alignment.Start,
+    ) {
+        Text(
+            text = "Override parameters",
+            style = MaterialTheme.typography.titleMedium,
+            modifier = Modifier.align(Alignment.CenterHorizontally)
+        )
+
+            Text(text = "Adjust basemap", style = MaterialTheme.typography.labelLarge)
+            Row(
```

Comment:
There seems to be an extra indentation here: 
```suggestion
        Text(
            text = "Override parameters",
            style = MaterialTheme.typography.titleMedium,
            modifier = Modifier.align(Alignment.CenterHorizontally)
        )

        Text(text = "Adjust basemap", style = MaterialTheme.typography.labelLarge)
        Row(
```